config/300-serving.yaml

```diff
@@ -12,278 +12,288 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: knativeservings.operator.knative.dev
   labels:
     operator.knative.dev/release: devel
 spec:
-  additionalPrinterColumns:
-  - JSONPath: .status.version
-    name: Version
-    type: string
-  - JSONPath: .status.conditions[?(@.type=="Ready")].status
-    name: Ready
-    type: string
-  - JSONPath: ".status.conditions[?(@.type==\"Ready\")].reason"
-    name: Reason
-    type: string
   group: operator.knative.dev
-  names:
-    kind: KnativeServing
-    listKind: KnativeServingList
-    plural: knativeservings
-    singular: knativeserving
-  preserveUnknownFields: false
-  scope: Namespaced
-  subresources:
-    status: {}
-  validation:
-    openAPIV3Schema:
-      type: object
-      description: Schema for the knativeservings API
-      properties:
-        apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation
-            of an object. Servers should convert recognized schemas to the latest
-            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
-          type: string
-        kind:
-          description: 'Kind is a string value representing the REST resource this
-            object represents. Servers may infer this from the endpoint the client
-            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
-          type: string
-        metadata:
-          type: object
-        spec:
-          description: Spec defines the desired state of KnativeServing
-          properties:
-            version:
-              description: The version of Knative Serving to be installed
-              type: string
-            manifests:
-              description: A list of serving manifests, which will be installed by the operator
-              type: array
-              items:
-                type: object
+  versions:
```

Comment:
The order of the attributes has been changed a lot comparing to the existing one. Is there any reason for that?